inference-engine/src/mkldnn_plugin/mkldnn_async_infer_request.cpp

```diff
@@ -8,10 +8,7 @@
 MKLDNNPlugin::MKLDNNAsyncInferRequest::MKLDNNAsyncInferRequest(const InferenceEngine::InferRequestInternal::Ptr& inferRequest,
                                                                const InferenceEngine::ITaskExecutor::Ptr& taskExecutor,
                                                                const InferenceEngine::ITaskExecutor::Ptr& callbackExecutor)
-        : InferenceEngine::AsyncInferRequestThreadSafeDefault(inferRequest, taskExecutor, callbackExecutor) {}
-
-void MKLDNNPlugin::MKLDNNAsyncInferRequest::Infer_ThreadUnsafe() {
-    InferUsingAsync();
```

Comment:
Default implementation is enough now for CPU plugin. Yes. The user thread is used as master thread for synchronous infernece

Reply:
Since other part of the MR are seemigly merged (as part of other PRs), this line seems to be the only item that remains from the PR. @apankratovantonp can we clean/rebase this PR and merge?

Reply:
@myshevts  What MR? The MR can be merged. What should be cleaned&